pages/common/doctl-account.md

```diff
@@ -0,0 +1,16 @@
+# doctl account
+
+> Retrieve information about digital ocean accounts.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/account/>.
+
+- Show available commands:
+
+`doctl account --help`
+
+- Show account info:
+
+`doctl account get`
+
+- Show the hourly API limit, progress towards it, and when the limit resets:
```

Comment:
```suggestion
- Show the hourly API limit, progress towards it, and when the rate limit resets:
```

What do you think about this suggestion, I feel the sentence reads a little strange without it.